packages/atlas/src/views/global/NftPurchaseBottomDrawer/NftPurchaseBottomDrawer.tsx

```diff
@@ -587,27 +583,42 @@ export const NftPurchaseBottomDrawer: FC = () => {
                 />
               )}
             </Row>
-            {(bid || type === 'buy_now') && (
+            {(bid || canBuyNow) && (
               <>
                 <Row>
                   <Text as="span" variant="t100" color="colorText">
                     Transaction fee
                   </Text>
-                  <NumberFormat as="span" value={transactionFee} withToken variant="t100" color="colorText" />
+                  {(canBuyNow ? !buyNowFee.toNumber() : !makeBidFee.toNumber()) ||
+                  buyNowFeeLoading ||
+                  makeBidFeeLoading ? (
+                    <SkeletonLoader width={80} height={16} />
+                  ) : (
+                    <NumberFormat as="span" value={transactionFee} withToken variant="t100" color="colorText" />
+                  )}
                 </Row>
                 <Row>
                   <Text as="span" variant="h500" color="colorText">
                     You will pay
                   </Text>
```

Comment:
In the design there is a short text: "You need to fill out the amount first" when the input is empty. You didn't include that. 
https://www.figma.com/file/wjeAOZq77pSUFZsYXJqKIB/NFT-purchase-view?node-id=1036%3A102849